src/libmusig2.h

```diff
@@ -9,16 +9,20 @@
 #include "random.h"
 
 
-// todo: use the secp library constants
-#define V                               2           // Number of nonce values
-#define SCALAR_BYTES                    32          // SCALAR BYTES
-#define XONLY_BYTES                     32          // XONLY PUBLIC KEY BYTES
-#define PAR_SIG_BYTES                   32          // PARTIAL SIGNATURE BYTES
-#define PK_BYTES                        64          // FULL SIZE PUBLIC KEY BYTES
-#define SCH_SIG_BYTES                   64          // SCHNORR SIGNATURE BYTES
-#define SER_PK_BYTES_COMPRESSED         33          // Serialized public key bytes for compressed version
+#define V                                   2           // Number of nonce values
+#define MUSIG2_SCALAR_BYTES                 32          // SCALAR BYTES
+#define MUSIG2_PARSIG_BYTES                 32          // PARTIAL SIGNATURE BYTES
+#define MUSIG2_PUBKEY_BYTES                 32          // XONLY PUBLIC KEY BYTES
+#define MUSIG2_PUBKEY_BYTES_COMPRESSED      33          // COMPRESSED PUBLIC KEY BYTES
+#define MUSIG2_PUBKEY_BYTES_FULL            64          // FULL SIZE PUBLIC KEY BYTES
```

Comment:
It's counterintuitive that `BYTES_COMPRESSED` is larger than `BYTES`. I would find a different name to the former (maybe something to do with omit sign bit or something of the sort)